modules/cnf-scheduling-numa-aware-workloads-with-manual-performance-setttings.adoc

```diff
@@ -0,0 +1,200 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-scheduling-numa-aware-workloads-with-manual-performance-setttings_{context}"]
+= Scheduling workloads with the NUMA-aware scheduler with manual performance settings
+
+You can schedule workloads with the NUMA-aware scheduler using `Deployment` CRs that specify the minimum required resources to process the workload.
+
+The following example deployment uses NUMA-aware scheduling for a sample workload.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Install the NUMA Resources Operator and deploy the NUMA-aware secondary scheduler.
+
+.Procedure
+
+. Get the name of the NUMA-aware scheduler that is deployed in the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get numaresourcesschedulers.nodetopology.openshift.io numaresourcesscheduler -o json | jq '.status.schedulerName'
+----
++
+.Example output
+[source,terminal]
+----
+topo-aware-scheduler
+----
+
+. Create a `Deployment` CR that uses scheduler named `topo-aware-scheduler`, for example:
+
+.. Save the following YAML in the `nro-deployment.yaml` file:
++
+[source,yaml]
+----
+apiVersion: apps/v1
```

Comment:
You can add:
securityContext: <securityContext> <2>

2. securityContext  refer to security context documentation

Reply:
this seems a bit overkill, let me review the example

Reply:
> Starting from 4.12 cluster will never schedule workload without proper SecurityContext. Unfortunatelly the example from our documentation'll not work. I don't insist we need to specify SecurityContext but at least we need to mention this fact. WDYT ?

Wan't the enforcing of PSA postponed to 4.14?

Reply:
@ffromani - any resolution on the above?

Reply:
not yet. Need to focus on other matters.

Reply:
Hi @ffromani - checking in on this again. 
FYI - docs freeze is in just over two weeks (April 12) so it would be good to start closing this one out soon.

Reply:
I'll have closure this week. From the current testing I've done so far the security context is nice to add but not yet mandatory, so I'm inclined to defer any update in this area to a knowledge base article.

Reply:
> I'll have closure this week. From the current testing I've done so far the security context is nice to add but not yet mandatory, so I'm inclined to defer any update in this area to a knowledge base article.

From what I can tell checking recent snapshots of both the NUMA-aware scheduler and builds of 4.13, I confirm this take. IMO it is OK to add this useful information in a knowledge base article and doesn't need to be in the reference docs just yet.